src/app/_components/_dev_BreakpointDebbuger.tsx

```diff
@@ -0,0 +1,12 @@
+export function BreakpointDebbuger() {
```

Comment:
```suggestion
export function BreakpointDebugger() {
```

Reply:
@CharlyMartin Minor spelling mistake left - other than that - ready to be merged. 🚀